src/debugAdapter/goDebug.ts

```diff
@@ -486,7 +486,7 @@ class Delve {
 		});
 	}
 
-	public callPromise<T>(command: string, args: any[]): Thenable<T> {
+	public async callPromise<T>(command: string, args: any[]): Promise<T> {
```

Comment:
We don;t need to mark a function as `async` if we don't plan to use `await` inside the function

Reply:
Just curious... What is the reason to convert the `Thenable` to `Promise`?

Reply:
According the [spec](https://github.com/Microsoft/vscode/blob/master/src/typings/thenable.d.ts#L7-L10) native promises are preferred. Since I needed to use await on the second eval call and assumed it was talking about typescript promises (forgive my unawareness of the node/JS conventions) I changed the promise returns in the adapter to follow suit

This module is isolated from the rest of the extension so I thought there would not be any incompatibilities or build problems; I ackowledge that making this change in the extension code requires a much larger effort, though

Reply:
I don't have any issues making this change, I was just curious as to what drove you to it